### PR TITLE
Configurable Dinner Tables Feature 

### PR DESCRIPTION
**Pull request by jlctmaster**
Configurable Dinner Tables Feature from @deepshah last change

### CODE REVIEW

**Comment by jlctmaster** [comment]
Hello @daN4cat , have you been able to validate the correct functioning of this new feature?

**Comment by daN4cat** [comment]
@jlctmaster can you please squash everything in one commit? There is a lot of duplicated history and many merge from master....

**Comment by daN4cat** [comment]
Also this feature should be something that can be enabled or disabled.
It's really specific to restaurants and not that applicable to most of OSPOS users that seem to be mainly shops.

**Comment by odiea** [comment]
He did add an app config in the tables config.

File: LICENSE [hunk 1]
```diff
@@ -11,7 +11,7 @@ Copyright (c) 2015 Aamir Shahzad (aka asakpke), RoshanTech.com
 Copyright (c) 2015 Toni Haryanto (aka yllumi)
 Copyright (c) 2016-2017 Ramkrishna Mondal (aka RamkrishnaMondal)
 Copyright (c) 2016 Rinaldy@dbarber (aka rnld26)
-Copyright (c) 2016 Jorge Colmenarez (aka jlctmaster), frontuari.com
+Copyright (c) 2016-2017 Jorge Colmenarez (aka jlctmaster), frontuari.com
```
Comment (daN4cat) [inline LICENSE]: Please add deepshah as well if not present already.

File: application/libraries/Sale_lib.php [hunk 2]
```diff
@@ -789,6 +814,12 @@ public function get_total()
 
 		return $total;
 	}
+
+    public function get_empty_tables()
```
Comment (daN4cat) [inline application/libraries/Sale_lib.php]: indentation issue, please use TABs

File: application/models/Dinner_table.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	public function get_name($dinner_table_id)
+	{
+		$this->db->from('dinner_tables');
+		$this->db->where('dinner_table_id',$dinner_table_id);
+
+		return $this->db->get()->row()->name;
+	}
+
+	public function get_all()
+    {
```
Comment (daN4cat) [inline application/models/Dinner_table.php]: there are indentation issues throughout this code unit

File: database/3.0_to_3.1.sql [hunk 4]
```diff
@@ -0,0 +1,31 @@
+--
```
Comment (daN4cat) [inline database/3.0_to_3.1.sql]: Please move this content to 3.0.2_to_3.1.0.sql and remove this file

File: database/database.sql [hunk 5]
```diff
@@ -514,10 +514,12 @@ CREATE TABLE `ospos_sales` (
   `comment` text NOT NULL,
```
Comment (daN4cat) [inline database/database.sql]: Please change tables.sql as that's the master used to create database.sql and phppos_migrate.sql when running grunt.

File: public/license/LICENSE [hunk 6]
```diff
@@ -11,7 +11,7 @@ Copyright (c) 2015 Aamir Shahzad (aka asakpke), RoshanTech.com
 Copyright (c) 2015 Toni Haryanto (aka yllumi)
```
Comment (daN4cat) [inline public/license/LICENSE]: Same comment as for the other file, please add deepshah name to the file.

**Comment by daN4cat** [comment]
@odiea indeed it seems so.

@jlctmaster together with the squash of the commits please see my review feedback.

**Comment by daN4cat** [comment]
@jlctmaster you also need to rebase as code in master just changed

**Comment by jlctmaster** [comment]
Hello @daN4cat , I prepare the corrections and apply the change to upload it.

**Comment by jlctmaster** [comment]
Hello @daN4cat , I'm going to close this PR and create a new one with the corrections / suggestions

**Comment by daN4cat** [comment]
Replaced by #1169

**Pull request closed without merging**